fv3gfs-physics/fv3gfs/physics/stencils/update_dwind_phys.py

```diff
@@ -164,13 +164,26 @@ class AGrid2DGridPhysics:
     Fortran name is update_dwinds_phys
     """
 
-    def __init__(self, stencil_factory: StencilFactory, grid, namelist, grid_info):
-        self.grid = grid
+    def __init__(
+        self,
+        stencil_factory: StencilFactory,
+        partitioner: TilePartitioner,
+        rank,
+        namelist,
+        grid_info,
```

Comment:
For my reference, what is grid_info and why is it everywhere?

Reply:
This is the missing grid info that is currently provided by serialized data. We just haven't gotten around using MetricTerms in physics, will do that in a separate PR.